src/Nethermind/Nethermind.JsonRpc/Modules/Trace/TraceRpcModule.cs

```diff
@@ -109,7 +108,7 @@ public ResultWrapper<IEnumerable<ParityTxTraceFromReplay>> trace_callMany(Transa
         /// </summary>
         public ResultWrapper<ParityTxTraceFromReplay> trace_rawTransaction(byte[] data, string[] traceTypes)
         {
-            Transaction tx = _txDecoder.Decode(new RlpStream(data), RlpBehaviors.SkipTypedWrapping);
+            Transaction tx = TxDecoder.Instance.Decode(new RlpStream(data), RlpBehaviors.SkipTypedWrapping);
```

Comment:
We can leave the field instead of inlining:

```suggestion
            private readonly TxDecoder _txDecoder => TxDecoder.Instance;
            // ...
            Transaction tx = _txDecoder.Decode(new RlpStream(data), RlpBehaviors.SkipTypedWrapping);
```

Reply:
Yes, and I switched all to lazy, looks better with it imo